addons/website_mass_mailing/static/src/js/website_mass_mailing.editor.js

```diff
@@ -42,75 +59,79 @@ var mass_mailing_common = options.Class.extend({
     popup_template_id: "editor_new_mailing_list_subscribe_popup",
     popup_title: _t("Add a Newsletter Subscribe Popup"),
-    select_mailing_list: function (previewMode, value) {
+
+    /**
+     * @override
+     */
+    start: function () {
         var self = this;
-        return this._super(previewMode, value).then(function (mailing_list_id) {
-            ajax.jsonRpc('/web/dataset/call', 'call', {
-                model: 'mail.mass_mailing.list',
-                method: 'read',
-                args: [[parseInt(mailing_list_id)], ['popup_content'], self.options.recordInfo.context],
-            }).then(function (data) {
-                self.$target.find(".o_popup_content_dev").empty();
-                if (data && data[0].popup_content) {
-                    $(data[0].popup_content).appendTo(self.$target.find(".o_popup_content_dev"));
-                }
-            });
+        this.$target.find('.o_edit_popup').on('click.newsletter_popup_option', function (ev) {
+            ev.stopPropagation();
+            self.$target.attr('quick-open', true);
+            self._refreshPublicWidgets();
+        });
+        this.$target.on('shown.bs.modal.newsletter_popup_option hide.bs.modal.newsletter_popup_option', function () {
+            self.$target.closest('.o_editable').trigger('content_changed');
+            self.trigger_up('deactivate_snippet');
         });
-    },
-});
-
-WysiwygMultizone.include({
-    events: _.extend({}, WysiwygMultizone.prototype.events, {
-        'click #edit_dialog': 'edit_dialog',
-        'click .o_popup_modal_content [data-dismiss="modal"]': 'close_dialog',
-    }),
-    save: function () {
-        var $target = $('#wrapwrap').find('#o_newsletter_popup');
-        if ($target && $target.length) {
-            this.close_dialog();
-            $('.o_popup_bounce_small').show();
-            if (!$target.find('.o_popup_content_dev').length) {
-                $target.find('.o_popup_modal_body').prepend($('<div class="o_popup_content_dev" data-oe-placeholder="' + _t("Type Here ...") + '"></div>'));
-            }
-            var content = $('#wrapwrap .o_popup_content_dev').html();
-            var newsletter_id = $target.parent().attr('data-list-id');
-            ajax.jsonRpc('/web/dataset/call', 'call', {
-                model: 'mail.mass_mailing.list',
-                method: 'write',
-                args: [
-                    parseInt(newsletter_id),
-                    {'popup_content':content},
-                    this.options.recordInfo.context,
-                ],
-            });
-        }
         return this._super.apply(this, arguments);
     },
+    /**
+     * @override
+     */
     destroy: function () {
-        this.close_dialog();
-        this._super();
+        this.$target.off('shown.bs.modal.newsletter_popup_option hide.bs.modal.newsletter_popup_option');
+        this.$target.find('.o_edit_popup').off('click.newsletter_popup_option');
```

Comment:
Just:
```javascript
this.$target.off('.newsletter_popup_option');
```
instead of these two lines.
That's why we use jQuery namespaces ;)

Reply:
thanks for the explanation :)